packages/expo-updates/e2e/fixtures/project_files/babel.config.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function (api) {
+  api.cache(true);
+  return {
+    presets: ['babel-preset-expo'],
+    plugins: [
+      [
+        'transform-inline-environment-variables',
```

Comment:
leaning toward to dynamically create a **.env** file with
```
EXPO_PUBLIC_UPDATES_SERVER_PORT=4747
```

then we can reference `process.env.EXPO_PUBLIC_UPDATES_SERVER_PORT` from jsx. good for dogfooding.

Reply:
I’ll see if I can make that work! Maestro seems to want env names to be prefixed with MAESTRO_